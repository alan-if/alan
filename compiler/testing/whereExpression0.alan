The l Isa location
  Has l l.
  Has c c.
End The l.

The co Isa object
  Container
End The co.

The ct Isa thing
  Container Taking thing.
End The ct.


Start At l.
  If l In l Then "l in l" End If.
