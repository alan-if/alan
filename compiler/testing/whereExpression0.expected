########## whereExpression0 ##########


whereExpression0.alan

    3.    Has c c.
=====>          1

  *1*   310 E : Identifier 'c' not defined.

   16.    If l In l Then "l in l" End If.
=====>            1

  *1*   318 E : The 'l' does not have the container property, which is required
                in Expression after IN.


        2 error(s).
        No detected warnings.
        1 informational message(s).

